routes/web.php

```diff
@@ -21,6 +22,8 @@
 
 require __DIR__.'/auth.php';
 
+Route::get('/supplier', [SupplierController::class, "index"])->middleware(['auth']);
+Route::get('/supplier/{id}', [SupplierController::class, "show"])->middleware(['auth']);
```

Comment:
these routes could be moved down to separate them from the PurchaseOrder routes as well as put them near the other supplier routes.

Reply:
I agree, changes have been made!